File: FC.Shared/ViewModels/Search/ArtistSearchResult.cs
using FC.Interfaces.ViewModels;
using FC.Shared.Entities;
using FC.Shared.EntityMapper;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FC.Shared.ViewModels.Search
{
    public class ArtistSearchResult : BaseModel
    {

        
        public string Country
        {
            get; set;
        }

        
        public DateTime Date
        {
            get; set;
        }

        public string Location { get; set; }

        
        public int[] Genres
        {
            get; set;
        }

        
        public string Image
        {
            get; set;
        }

        
        public string Name
        {
            get; set;
        }

        private string type = "artist";
        public string Type
        {
            get
            {
                return type;
            }
            set
            {
                type = value;
            }
        }
    }
}

File: FC.WebMVC/Scripts/Shared/ViewModels/Artist/ArtistSearchResult.ts
module FC.Shared.ViewModels {
    import CM = FC.Core.CoreModel;
    import INT = FC.Shared.Interfaces;
    import MODELS = FC.Shared.Models;
    import MODULES = FC.Modules;
    export class ArtistSearchResult implements INT.ISearchResult {
        UmbracoID: number;
        Country: string;
        Name: string;
        Genres: Array<number>;
        Image: string;
        Type: string;
        Date: Date;
        Location: string;
    }
}